docs/guides/postgres/monitoring/using-builtin-prometheus.md

```diff
@@ -69,52 +68,85 @@ Run following command to create example above.
 
 ```console
-$ kubectl describe svc -n demo builtin-prom-postgres
-Name:              builtin-prom-postgres
-Namespace:         demo
-Labels:            kubedb.com/kind=Postgres
-                   kubedb.com/name=builtin-prom-postgres
-Annotations:       monitoring.appscode.com/agent=prometheus.io/builtin
-                   prometheus.io/path=/kubedb.com/v1alpha1/namespaces/demo/postgreses/builtin-prom-postgres/metrics
-                   prometheus.io/port=56790
-                   prometheus.io/scrape=true
-Selector:          kubedb.com/kind=Postgres,kubedb.com/name=builtin-prom-postgres
-Type:              ClusterIP
-IP:                10.107.124.174
-Port:              prom-http  56790/TCP
-TargetPort:        %!d(string=prom-http)/TCP
-Endpoints:         172.17.0.8:56790
-Session Affinity:  None
+$ kubectl get svc -n demo
+NAME                             TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)     AGE
+builtin-prom-postgres            ClusterIP   10.101.93.0      <none>        5432/TCP    1m
+builtin-prom-postgres-replicas   ClusterIP   10.105.183.227   <none>        5432/TCP    1m
+builtin-prom-postgres-stats      ClusterIP   10.107.173.32    <none>        56790/TCP   46s
+kubedb                           ClusterIP   None             <none>        <none>      1h
+```
+
+Let's describe Service `builtin-prom-postgres-stats`
+
+```console
+$ kubedb describe svc -n demo builtin-prom-postgres-stats
+Name:         builtin-prom-postgres-stats
+Namespace:    demo
+Labels:       kubedb.com/kind=Postgres
+              kubedb.com/name=builtin-prom-postgres
+Annotations:  monitoring.appscode.com/agent=prometheus.io/builtin
+              prometheus.io/path=/kubedb.com/v1alpha1/namespaces/demo/postgreses/builtin-prom-postgres/metrics
+              prometheus.io/port=56790
+              prometheus.io/scrape=true
+API Version:  v1
+Kind:         Service
+Metadata:
+  Creation Timestamp:  2018-09-24T11:56:38Z
+  Owner References:
+    API Version:           kubedb.com/v1alpha1
+    Block Owner Deletion:  false
+    Kind:                  Postgres
+    Name:                  builtin-prom-postgres
+    UID:                   c077d83a-bff0-11e8-ad46-e6638755530c
+  Resource Version:        47773
+  Self Link:               /api/v1/namespaces/demo/services/builtin-prom-postgres-stats
+  UID:                     e4f47ef9-bff0-11e8-ad46-e6638755530c
+Spec:
+  Cluster IP:  10.107.173.32
+  Ports:
+    Name:         prom-http
+    Port:         56790
+    Protocol:     TCP
+    Target Port:  prom-http
+  Selector:
+    Kubedb . Com / Kind:  Postgres
+    Kubedb . Com / Name:  builtin-prom-postgres
+  Session Affinity:       None
+  Type:                   ClusterIP
+Status:
+  Load Balancer:
+Events:  <none>
+
 ```
 
 You can see that the service contains following annotations.
 
 ```console
+monitoring.appscode.com/agent=prometheus.io/builtin
```

Comment:
This annotation is not a specification of prometheus server. Instead, This `monitoring.appscode.com/agent=prometheus.io/builtin` annotation is used from KubeDB operator. So, let's not add that here.

Reply:
removed